lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_auth_client.rb

```diff
@@ -6,26 +6,42 @@ module FirebaseAppDistributionAuthClient
       TOKEN_CREDENTIAL_URI = "https://oauth2.googleapis.com/token"
 
       def fetch_auth_token(google_service_path)
+        if ENV["XDG_CONFIG_HOME"]
+          config_path = File.expand_path("configstore/firebase-tools.json", ENV["XDG_CONFIG_HOME"])
+        else
+          config_path = File.expand_path(".config/configstore/firebase-tools.json", "~")
+        end
         if !google_service_path.nil? && !google_service_path.empty?
           service_account(google_service_path)
         elsif ENV["FIREBASE_TOKEN"]
-          firebase_token
+          firebase_token(ENV["FIREBASE_TOKEN"])
```

Comment:
Nice 👍

Reply:
Oh, I thought this was for style but it was essentially a refactor to support the new auth method. Still like it!